packages/tools/pluggable-widgets-tools/configs/rollup-plugin-widget-typing.js

```diff
@@ -23,6 +23,10 @@ export function widgetTyping({ sourceDir }) {
             if (!firstRun) {
                 await runTransformation(sourceDir);
             }
+            await execShellCommand(
+                `npx pluggable-widgets-tools format:custom-paths -- "../typings/**/*.ts"`,
```

Comment:
Can we be more strict and only format the file generated by this function? You have the output, so its easy to reuse it.

Reply:
Good one. You could let the `transformPackage` function return the destination path and use that to format the generated typing file.